shared/helpers.js

```diff
@@ -50,8 +50,42 @@ const fetchThemeBundle = async (obj, renderer, theme)=>{
 	}));
 };
 
+let pageArray = [];
```

Comment:
What is the purpose of the `pageArray` over simply setting the current page directly when the `intersectionObserver` fires?

Reply:
My thinking was that - particularly with the new BrewRenderer toolbar - that there can be multiple pages visible in the view port, so an array of page numbers is more appropriate than a single value.